### DISCUSSION TITLE
Is it possibilke to save without triggering new deploy?

### DISCUSSION THREAD

**Discussion by zanhk** [Thinkmill/keystatic#1023] [Q&A]
Just reposting from discord https://discord.com/channels/1139017922073866332/1139017922895937598/1218541058913341460:

Is there a way to "publish" the site only after saving bunch of changes instead of triggering a new commit every time i save?
what I mean is that every time I make a change and save, keystatic make a commit with [skip-ci] https://developers.cloudflare.com/pages/configuration/branch-build-controls/#skip-builds in the commit message,  then it show a "publish" that just trigger an empty commit.
For showing the "publish" button either look at the last commit or save a flag on local storage (maybe?)

![image](https://github.com/Thinkmill/keystatic/assets/17980844/6f0e381c-7747-4062-915e-aa5f7aff55c1)

![image](https://github.com/Thinkmill/keystatic/assets/17980844/6519d33c-e50c-496d-b747-0180a9cbdac2)

the problem is that one: the customer make a change and save, he make 30 changes and trigger 30 deploys, the new website is available in hours (x * 3min)

**Comment by arkrosclou** [comment]
functionally, it would like something similar to how pool request works, all changes in the admin are noticed by badges and the user has to make a single delivery when he is ready, but it is not always convenient for the user to do a pool request to GitHub, so mechanism described above it would be nice
